layers/+tools/lsp/funcs.el

```diff
@@ -87,11 +87,32 @@ https://github.com/emacs-lsp/lsp-javascript/issues/9#issuecomment-379515379"
     "Ts" #'lsp-ui-sideline-mode
     "TF" #'spacemacs/lsp-ui-doc-func
     "TS" #'spacemacs/lsp-ui-sideline-symb
-    "TI" #'spacemacs/lsp-ui-sideline-ignore-duplicate
-    )
+    "TI" #'spacemacs/lsp-ui-sideline-ignore-duplicate))
 
-  )
+(defun spacemacs//lsp-bind-navigation-keys-for-mode (mode)
+  (ecase lsp-navigation
+    ('simple (spacemacs//lsp-bind-simple-navigation-functions-for-mode mode "g"))
+    ('peek (spacemacs//lsp-bind-peek-navigation-functions-for-mode mode "g"))
+    ('both
+      (spacemacs//lsp-bind-simple-navigation-functions-for-mode mode "g")
+      (spacemacs//lsp-bind-peek-navigation-functions-for-mode mode "p"))))
```

Comment:
Honestly I would prefer the `'both` option to not exist and only have the `'simple` or `'peek` options under `g`. This way `p` can be used by other functionality.

Another option would be lowercase for simple and uppercase for peek, but that might be confusing as well... What do you say?

Reply:
I use both. `g` when I know where I'm going and `p` when I'm not sure -- the context given by the lsp-ui-peek overlay is very useful when using e.g. find-callers / references type lookup functions.
Though could certainly live with a different prefix (i.e. capital G as you suggest). So could go that route and either eliminate the preference config value altogether (i.e. always bind both sets of functions under g and G respectively) or rename the config var to `lsp-navigation-invert-bindings` or something -- when non-nil, bind the lsp-ui-peek variants under `g` and move the xref variants to `G`.

Any views from contributors to other lsp-derived layers?  @yyoncho , @jamartin9, @ljupchokotev, @ztlevi (based on a quick search for recent PRs mentioning lsp)

Reply:
@cormacc I use both as well, mainly xref ones so I prefer them as a default. In lsp java layer which is not synched with lsp base layer I am using `mgr` and `mgR` for UI peek but `mGr`  looks better.

Reply:
OK - went with the simplest change -- just replaced 'p' with 'G'. Works better alright -- easy to remember. Rebased/amended/pushed.

Reply:
@sdwolfz  Had done `git commit --amend` rather than `git commit -a --amend` <facepalm>, so change wasn't pushed earlier -- fixed now.